packages/react-compass/src/drawer/drawer.styles.ts

```diff
@@ -0,0 +1,157 @@
+import {styled} from '../theme'
+import {keyframes} from '../theme/stitches.config'
+import type {VariantProps} from '../utils/stitches.types'
+
+const slideInLeft = keyframes({
+  from: {transform: 'translateX(-100%)'},
+})
+
+const slideOutLeft = keyframes({
+  to: {transform: 'translateX(-100%)'},
+})
+
+const slideInRight = keyframes({
+  from: {transform: 'translateX(100%)'},
+})
+
+const slideOutRight = keyframes({
+  to: {transform: 'translateX(100%)'},
+})
+
+const slideInUp = keyframes({
+  from: {transform: 'translateY(100%)'},
+})
+
+const slideOutUp = keyframes({
+  to: {transform: 'translateY(100%)'},
+})
+
+export const StyledDrawer = styled('dialog', {
+  padding: 0,
+  insetBlock: 0,
+
+  maxHeight: '100%',
+  maxWidth: 'unset',
+
+  flexDirection: 'column',
+
+  border: 'none',
+  boxSizing: 'border-box',
+  zIndex: '50',
```

Comment:
i think z-index needs to be less than 49
https://www.figma.com/file/MQVnynY3v1pI55Cm6l5qHK/CDGNotes?type=design&node-id=1-63&mode=design&t=5zW5qLdhfzUFBZ2R-0

Reply:
@kaiceresy let's test with modal case, i think modal will be above drawer